backend/CHANGELOG.rst

```diff
@@ -7,6 +7,7 @@ This file keeps track of all notable changes to license-manager-backend
 Unreleased
 ----------
 * Add cluster_id column to config table to identify which cluster the configuration applies to
+* Created new route to fetch configurations filtering by the cluster requesting the data
```

Comment:
I am not sure what this means. Is the following correct?
```suggestion
* Added new route to fetch all configuration from a specific cluster
```

Reply:
Yes. This route is going to be used only by the agents. It identifies which cluster is requesting the data and automatically filters the configs.

Reply:
I changed it in the latest commit to use your suggestion, thanks!